repository/c.json

```diff
@@ -2067,6 +2067,17 @@
 			]
 		},
 		{
+			"name": "Color Schemes - Wicked Schemes by b0xw00d",
+			"details": "https://github.com/b0xw00d/wicked_color_schemes",
+			"labels": ["theme", "color scheme", "syntax highlighting", "wicked"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true
```

Comment:
You'll need to add at least one [semver](http://semver.org) tag to https://github.com/b0xw00d/wicked_color_schemes/tags for this to show up to users.